shared/chat/conversation/list-area/normal/index.desktop.js

```diff
@@ -360,16 +360,16 @@ class Thread extends React.PureComponent<Props, State> {
   _onResize = ({scroll}) => {
     if (this._scrollHeight) {
       // if the size changes adjust our scrolltop
-      const list = this._listRef.current
-      if (list) {
-        this._logAll(list, '_onResize', () => {
-          this._ignoreScrollToBottomRefCount++
-          if (this.state.isLockedToBottom) {
-            list.scrollTop = list.scrollTop + scroll.height - list.clientHeight
```

Comment:
two bugs here:

1) `list.scrollTop` should be _set_ to `scroll.height - list.clientHeight`, not added together
2) `scroll.height` is the height of the inner div, which is different -- `list.scrollHeight` should be used instead.

`_scrollToBottom` already does the right thing so call that.

This bug probably didn't do anything, since `list.scrollTop` is capped anyway. But it confused me for a while.